src/navigation/RootStackNavigation.tsx

```diff
@@ -268,7 +270,7 @@ const AppUnlockHandler = ({ children }: { children: ReactNode }) => {
     const handleAppStateChange = (nextAppState: AppStateStatus) => {
       if (nextAppState === 'background' && walletMnemonic && !isCameraOpen) {
         loadBiometricsSettings().then((isBioEnabled) =>
-          navigation.navigate(isBioEnabled ? 'LandingScreen' : 'LoginWithPinScreen')
+          resetNavigation(navigation, isBioEnabled ? 'LandingScreen' : 'LoginWithPinScreen')
```

Comment:
I had to reset and not navigate, because if the user cancels the bio auth prompt, then they can swipe "back" and go to the previous screen. Resetting fixes it.